src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -80,7 +80,25 @@ public static AuthenticationBuilder AddScriptJwtBearer(this AuthenticationBuilde
                             }
                         });
 
-        private static TokenValidationParameters CreateTokenValidationParameters()
+        private static string[] GetValidAudiences()
```

Comment:
We should add test coverage for this. The source PR for v4 added test coverage for this. Can we do the same?

Reply:
On it

Reply:
Added the unit test 👍